src/rove_slam/package.xml

```diff
@@ -3,10 +3,15 @@
 <package format="3">
   <name>rove_slam</name>
   <version>0.0.0</version>
-  <description>TODO: Package description</description>
-  <maintainer email="simonroy99@hotmail.ca">rove</maintainer>
```

Comment:
Bruh

Reply:
Yeah, when I created the package using the ROS command, it auto-filled up those parameter